external/implementations/samlconf-openam-impl/src/main/java/org/codice/compliance/saml/plugin/openam/OpenAMDataParser.java

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+package org.codice.compliance.saml.plugin.openam;
+
+import static org.codice.compliance.CommonKt.USER_LOGIN;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Iterables;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+class OpenAMDataParser {
```

Comment:
❓ Is this class only invoked from within this package?

Reply:
Yep, the only class that uses it is the `OpenAMIdpSSOResponderProvider`